### ISSUE TITLE
Attention - this Library is not FREE for commercial use

### ISSUE THREAD

**Issue by matan-d** [recoilme/freemp#21]
According to the BASS Licence seen here:
http://www.un4seen.com/

*I dont work there, just wanted to let you guys know and also for other devs who comes across and are planning on using this code for commercial usage..

**Comment by recoilme** [comment]
Yes, you right. But license for commercial usage is cheap
https://www.un4seen.com/bass.html#license

**Comment by matan-d** [comment]
Yes you are right, it is not expensive and only needed when selling your app implementing this library..just wanted to let dev know before implementation.

The BASS library is actually amazingly powerful and stable!
